internal/billing/command_promo_list_test.go

```diff
@@ -5,14 +5,16 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/require"
+
+	"github.com/confluentinc/cli/v3/pkg/billing"
 )
 
 func TestFormatBalance(t *testing.T) {
 	require.Equal(t, "$0.00/1.00 USD", formatBalance(0, 10000))
 }
 
 func TestConvertToUSD(t *testing.T) {
-	require.Equal(t, 1.23, ConvertToUSD(12300))
+	require.Equal(t, 1.23, billing.ConvertToUSD(12300))
 }
```

Comment:
Let's make sure we also move this test into `pkg`.

Reply:
Oops, good call